region/crateria/west/Green Pirates Shaft.json

```diff
@@ -267,6 +313,46 @@
         "Fix the camera by shooting the floor shot block and jumping down."
       ]
     },
+    {
+      "link": [2, 9],
+      "name": "G-Mode Morph Crystal Flash, Reserve Trigger, Up to the Beetoms",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphCrystalFlash",
+        {"autoReserveTrigger": {}},
+        {"or": [
+          {"enemyKill": {
+            "enemies": [["Green Space Pirate (standing)"]]
+          }},
+          {"enemyDamage": {
+            "enemy": "Green Space Pirate (standing)",
+            "type": "laser",
+            "hits": 3
+          }}
+        ]}
+      ],
+      "note": [
+        "Crystal Flash then damage down on the Pirates to Reserve trigger to exit G-Mode.",
```

Comment:
The goal of the strat is to fill your reserves and gain up to 400 health?

Reply:
Yes, in order to do Beetom shenanigans. Although I guess this needs to model the PB blocks being broken, otherwise everything requires Morph to get back down from 9 except the Beetom x-ray climb